src/main.jl

```diff
@@ -148,8 +148,8 @@ function SolverCore.solve!(
     [Int64, T, T, T, String, T, T, T],
   )
   verbose > 0 && @info log_row(Any[iter, ft, norm_∇f, 0.0, "First iteration", α])
-
-  callback(nlp, solver, stats)
```

Comment:
You can actually access the nlp_at_x by solver.stp.nlp_at_x , so I prefer `callback(nlp, solver, stats)`